partcad/src/partcad/utils_ocp.py

```
@@ -0,0 +1,111 @@
+
+# Some pieces of code in this file are taken from the ocp-tessellate project:
+#   https://github.com/bernhard-42/ocp-tessellate/
+#
+#   Copyright 2023 Bernhard Walter
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+
+from typing import Tuple, List
```

Ruff (0.8.2)

`typing.Tuple` is deprecated, use `tuple` instead

(UP035)